components/MDXComponents.tsx

```diff
@@ -41,14 +41,15 @@ const LinkCustom = (props) => {
 
     if (isInternalLink) {
         return (
-            <Link href={href}>
-                <a>{props.children}</a>
+            // @ts-ignore
```

Comment:
why ts-ignore ?

Reply:
Because then it gives a type error for Next/Link